eris_test.go

```diff
@@ -76,61 +109,63 @@ func TestErrorWrapping(t *testing.T) {
-			cause:  errors.New("external error"),
-			output: eris.New("external error"),
+			cause: errors.New("external error"),
+			output: []string{
+				"external error",
+			},
 		},
 		"standard error wrapping with a third-party root cause (errors.New)": {
-			cause:  errors.New("external error"),
-			input:  []string{"additional context", "even more context"},
-			output: eris.Wrap(eris.Wrap(eris.New("external error"), "additional context"), "even more context"),
+			cause: errors.New("external error"),
+			input: []string{"additional context", "even more context"},
+			output: []string{
+				"even more context: additional context: external error",
+				"additional context: external error",
+				"external error",
+			},
 		},
 		"wrapping a wrapped third-party root cause (errors.New and fmt.Errorf)": {
-			cause:  fmt.Errorf("additional context: %w", errors.New("external error")),
-			input:  []string{"even more context"},
-			output: eris.Wrap(eris.Wrap(eris.New("external error"), "additional context"), "even more context"),
+			cause: fmt.Errorf("additional context: %w", errors.New("external error")),
+			input: []string{"even more context"},
+			output: []string{
+				"even more context: additional context: external error",
+				"additional context: external error",
+				"external error",
+			},
 		},
 		"wrapping a wrapped third-party root cause (multiple layers)": {
-			cause:  fmt.Errorf("even more context: %w", fmt.Errorf("additional context: %w", errors.New("external error"))),
-			input:  []string{"way too much context"},
-			output: eris.Wrap(eris.Wrap(eris.Wrap(eris.New("external error"), "additional context"), "even more context"), "way too much context"),
+			cause: fmt.Errorf("even more context: %w", fmt.Errorf("additional context: %w", errors.New("external error"))),
+			input: []string{"way too much context"},
+			output: []string{
+				"way too much context: even more context: additional context: external error",
+				"even more context: additional context: external error",
+				"additional context: external error",
+				"external error",
+			},
 		},
 		"wrapping a wrapped third-party root cause that contains an empty layer": {
-			cause:  fmt.Errorf(": %w", errors.New("external error")),
-			input:  []string{"even more context"},
-			output: eris.Wrap(eris.Wrap(eris.New("external error"), ""), "even more context"),
+			cause: fmt.Errorf(": %w", errors.New("external error")),
+			input: []string{"even more context"},
+			output: []string{
+				"even more context: : external error",
+				": external error",
+				"external error",
+			},
 		},
 		"wrapping a wrapped third-party root cause that contains an empty layer without a delimiter": {
-			cause:  fmt.Errorf(": %w", errors.New("external error")),
-			input:  []string{"even more context"},
-			output: eris.Wrap(eris.Wrap(eris.New("external error"), ""), "even more context"),
+			cause: fmt.Errorf(": %w", errors.New("external error")),
```

Comment:
what delimiter? the test case above and this one are same, aren't they÷

Reply:
hmm yeah, you're right. i'll see if i can modify it to match the description or i'll remove it if that doesn't work

Reply:
fixed it